src/core/ui/components/user_account_settings/screens/summary.tsx

```diff
@@ -0,0 +1,230 @@
+    const connections = accountSettings?.connections || [];
+
+    return {
+      emails: connections.filter(
+        (connection) => connection.displayType === 'email'
+      ),
+      web3Wallets: connections.filter(
+        (connection) => connection.displayType === 'eth_wallet'
+      ),
+      web2Accounts: connections.filter(
+        (connections) => connections.displayType === 'social'
+      ),
+    };
+  }, [accountSettings?.connections]);
+
+  const nonCustodialWallets = useMemo(
+    () =>
+      web3Wallets.filter(
+        (connection) => connection.connectionType !== 'managed_wallet'
+      ),
+    [web3Wallets]
+  );
+
+  return accountSettings ? (
+    <>
+      <Header
+        title="Account Settings"
+        description="Adjust your account settings and connected social accounts here."
+      />
+      <Text component="h2" align={Align.Left} className={margin.top8}>
+        Profile Image
+      </Text>
+      <Avatar
+        className={margin.top2}
+        src={accountSettings.defaultProfileImage}
+        onClick={editProfilePicture}
+      />
+      <Text
+        component="h2"
+        align={Align.Left}
+        className={classNames(margin.top12, margin.bottom2)}
+      >
+        Name
+      </Text>
+      <div className={styles.action} onClick={() => router.navigate('name')}>
+        {accountSettings.name ? (
+          <Text align={Align.Left} className={classNames(styles.settingsValue)}>
+            {accountSettings.name}
+          </Text>
+        ) : (
+          <Text className={classNames(styles.addButton)} align={Align.Left}>
+            + Add name
+          </Text>
+        )}
+        <span className={styles.navigateAction}>
+          <svg
+            width="18"
+            height="14"
+            viewBox="0 0 18 14"
+            fill="none"
+            xmlns="http://www.w3.org/2000/svg"
+          >
+            <path
+              d="M10.6667 1.16667L16.5 7.00001M16.5 7.00001L10.6667 12.8333M16.5 7.00001L1.5 7"
+              stroke="#2F5FFC"
+              strokeWidth="2"
+              strokeLinecap="round"
+              strokeLinejoin="round"
+            />
+          </svg>
+        </span>
+      </div>
+      <Text
+        component="h2"
+        align={Align.Left}
+        className={classNames(margin.top12, margin.bottom2)}
+      >
+        Email address
+      </Text>
+      {emails.length ? (
+        emails.map((connection) => (
+          <div className={styles.actionOnHover} key={connection.id}>
+            <div style={{ display: 'flex', alignItems: 'center' }}>
+              <ConnectionTypeIcon type={connection.connectionType} />
+              <Text
+                align={Align.Left}
+                className={classNames(styles.settingsValue)}
+              >
+                {connection.displayValue}
+              </Text>
+              <Chip>Verified</Chip>
+            </div>
+            <BinIcon onClick={() => removeConnection(connection.id)} />
```

Comment:
Remember to not show the trash icon if the item cannot be removed (currently it does)